src/main/java/seedu/address/logic/commands/patients/EditPatientDetailsCommand.java

```diff
@@ -42,6 +42,8 @@
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Patient: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This patient has already been registered.";
+    public static final String MESSAGE_PATIENT_IN_QUEUE = "You cannot edit while patient is in queue.";
+    public static final String MESSAGE_PATIENT_BEING_SERVED = "You cannot edit while patient is being served.";
```

Comment:
Please remove the "You" in the error messages

Reply:
Fixed